app/src/main/AndroidManifest.xml

```diff
@@ -23,7 +23,11 @@
         <activity android:name=".RegisterActivity" />
         <activity android:name=".TeacherLoginActivity" />
         <activity android:name=".StudentLoginActivity" />
-
+        <activity android:name=".TeacherDashboard.ClassHomeActivity" />
+        <activity android:name=".LoginSelectionActivity.TeacherDashboard.TeacherDashboardActivity" />
+        <activity android:name=".TeacherDashboard.CreateStudentAccountActivity" />
+        <activity android:name=".TeacherDashboard.activityOverviewActivity" />
+        <activity android:name=".StudentDashboardActivity"></activity>
```

Comment:
You can remove the </activity> and replace the trailing > with /> as in the lines above this one.